OloEngine/tests/TestTempDir.cpp

```
@@ -0,0 +1,293 @@
+            {
+                ADD_FAILURE() << message;
+            }
+            else
+            {
+                (void)std::fprintf(stderr, "[TestTempDir] %s\n", message.c_str());
+            }
+        }
+        return dir;
+    }
+
+    std::filesystem::path TempFile(std::string_view name)
```

Cppcheck (2.21.0)

[style] The function 'TempFile' is never used.

(unusedFunction)